
Remove libjpeg path as ${libjpeg_prefix} is lost which results in -L/lib
which then fails Q/A check

Pending

diff -ur libgdiplus-4.2.org/configure.ac libgdiplus-4.2/configure.ac
--- libgdiplus-4.2.org/configure.ac	2015-12-18 15:30:24.000000000 +0100
+++ libgdiplus-4.2/configure.ac	2016-12-19 19:07:17.284548584 +0100
@@ -223,7 +223,7 @@
         jpeg_ok=no)
       AC_MSG_RESULT($jpeg_ok)
       if test "$jpeg_ok" = yes; then
-        JPEG='jpeg'; LIBJPEG='-L${libjpeg_prefix}/lib -ljpeg'
+        JPEG='jpeg'; LIBJPEG='-ljpeg'
       else
         AC_MSG_WARN(*** JPEG loader will not be built (JPEG header file not found) ***)
       fi
